README.md

```diff
@@ -157,6 +157,31 @@ Contributions to Manim are always welcome. In particular, there is a dire need f
 
 Most developers on the project use [Poetry](https://python-poetry.org/docs/) for management. You'll want to have poetry installed and available in your environment. You can learn more about `poetry` and how to use it at its [documentation](https://docs.manim.community/en/stable/installation/for_dev.html).
 
+## How to Cite Manim
+
+We acknowledge the importance of good software to support research, and we note
+that research becomes more valuable when it is communicated effectively. To
+demonstrate the value of Manim, we ask that you cite Manim in your work.
+Currently, the best way to cite Manim is to reference the
+[Manim home page](https://www.manim.community) using this BibTeX entry (the
+entry is for release `v0.6.0`, but can be adapted easily):
+
+```
+@Manual{Manim:v0.6.0,
+  key =          {Manim},
+  author =       {{The Manim Community Developers}},
+  title =        {{Manim} {M}athematical {A}nimation {F}ramework ({V}ersion v0.6.0)},
+  note =         {\url{https://www.manim.community}},
+  year =         2021,
+}
+```
+
+This should render a reference that looks more or less like this:
+
+42. The Manim Community Developers,
+    [Manim Mathematical Animation Framework (Version v0.6.0)](https://www.manim.community).
```

Comment:
I think a comma should come after (website) right?

Reply:
See https://benjamin-hackl.at/publications/#software -- it depends on the BibTeX style used; on my website it renders without comma.